sourcetracker/_sourcetracker.py

```diff
@@ -570,17 +548,18 @@ def gibbs_sampler(sink, cp, restarts, draws_per_restart, burnin, delay):
     for restart in range(restarts):
```

Comment:
btw, if this is a performance bottleneck, this would probably greatly benefit from cython